ss/ss.go

```diff
@@ -91,25 +60,30 @@ func splitByLines(bytes []byte) []string {
 	return strings.Split(str, "\n")
 }
 
-func toComDetails(ssOutput []string, protocol string, node *corev1.Node) ([]types.ComDetails, error) {
+func toComDetails(debugPod *debug.DebugPod, ssOutput []string, protocol string, node *corev1.Node) []types.ComDetails {
 	res := make([]types.ComDetails, 0)
 	nodeRoles := nodes.GetRole(node)
 
 	for _, ssEntry := range ssOutput {
-		cd, err := parseComDetail(ssEntry)
+		cd := parseComDetail(ssEntry)
+
+		name, err := getContainerName(debugPod, ssEntry)
 		if err != nil {
-			return nil, err
+			log.Debugf("failed to identify container for ss entry: %serr: %s", ssEntry, err)
 		}
+
+		cd.Container = name
 		cd.Protocol = protocol
 		cd.NodeRole = nodeRoles
 		cd.Optional = false
 		res = append(res, *cd)
 	}
 
-	return res, nil
+	return res
 }
 
-func identifyContainerForPort(debugPod *debug.DebugPod, ssEntry string) (string, error) {
+// getContainerName receives an ss entry and gets the name of the container exposing this port.
```

Comment:
nit: "ss entry" shows up a lot of times in this file, it would be useful to document how it looks like

Reply:
added an example of ss entry to the README